job-pool/src/Control/Concurrent/JobPool.hs

```diff
@@ -3,7 +3,7 @@
 {-# LANGUAGE NamedFieldPuns #-}
 
 
-module Network.Mux.JobPool (
+module Control.Concurrent.JobPool (
```

Comment:
Splitting things up in separate packages is generally a good thing, but does this 80 line module really deserve a separate package? Will it grow much in the future?

The list of packages is already quite long and there are so many places (CI stuff, `cabal.project` in downstream repos, ...) mentioning the packages that can be forgotten to update.

Reply:
> The list of packages is already quite long and there are so many places (CI stuff, cabal.project in downstream repos, ...) mentioning the packages that can be forgotten to update.

Updating the hashes in `cabal.project` is a valid concern, though we have `cardano-repo-tool` for that, so I am not worried about it.  And probably sooner than later we will move to `cabal-3.2` where speciyfing hashes will be easier (just one per repo, rather than one per _cabal project_).  

The problem with this module is that exporting it from  `network-mux` does not seem right.  It would be a nice case for a public library in `network-mux`, but this is not available.

Another option is to add it to `io-sim-classes`, though this is even worse ides.

From all the available options, probably moving it back (but still keeping it under `Control.Concurrent` scope) is the least wrong option.